metadata-ingestion/src/datahub/entrypoints.py

```diff
@@ -120,11 +129,22 @@ def init() -> None:
     host = click.prompt(
         "Enter your DataHub host", type=str, default="http://localhost:8080"
     )
-    token = click.prompt(
-        "Enter your DataHub access token (Supports env vars via `{VAR_NAME}` syntax)",
-        type=str,
-        default="",
-    )
+    host = ensure_valid_gms_url(host)
+    if use_password:
```

Comment:
eventually, I think we will want to prompt the user for token vs password instead of needing a `--use-password` flag

Reply:
yeah this is mainly for admin users convenience. Most companies will be using SSO for login and won't have passwords so I do not foresee that being a default need for most users of the CLI